### PR TITLE
v3.2.0 release

### PR DESCRIPTION
**Pull request by wlthomson**
## Release version

v3.2.0.

## Discussion

#1675.

## Change summary

- various bug fixes.
- adds core indicators functionality.
- adds dashboard functionality.

### CODE REVIEW

File: src/database/schema.js [hunk 1]
```diff
@@ -124,36 +140,41 @@ User.schema = {
...
+    SyncOut,
+    Transaction,
+    TransactionBatch,
+    TransactionCategory,
+    TransactionItem,
     Unit,
+    User,
   ],
-  schemaVersion: 10,
+  schemaVersion: 13,
```
Comment (joshxg) [inline src/database/schema.js]: think we should change this to 11

File: src/database/utilities/ReportDemoData.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+export const testData = [
```
Comment (joshxg) [inline src/database/utilities/ReportDemoData.js]: can we delete this before it goes into master sorry

File: src/database/schema.js (hunk 1 above)
Comment (wlthomson) [reply src/database/schema.js]: a7bbf5d63170944846942d5e0da84b2f89536d52.

File: src/database/utilities/ReportDemoData.js (hunk 2 above)
Comment (wlthomson) [reply src/database/utilities/ReportDemoData.js]: 5c3911f7bbc5aa33f9eaf56a0286a74a05a597d9.

**Comment by joshxg** [comment]
🎉 🎉

**Pull request merged**